source/command-monitoring/tests/find.yml

```diff
@@ -247,6 +248,76 @@ tests:
               nextBatch:
                 - { _id: 4, x: 44 }
           command_name: "getMore"
+  -
+    description: "A successful find event with a multiple getMore (>= 5.0)"
+    ignore_if_server_version_less_than: "5.0"
+    ignore_if_topology_type:
+      - "sharded"
+    operation:
+      name: "find"
+      arguments:
+        filter: { _id: { $gte: 1 }}
+        sort: { _id: 1 }
+        batchSize: {"$numberLong": "3"}
+        limit: {"$numberLong": "4"}
+    expectations:
+      -
+        command_started_event:
+          command:
+            find: *collection_name
+            filter: { _id: { $gte : 1 }}
+            sort: { _id: 1 }
+            batchSize: {"$numberLong": "3"}
+            limit: {"$numberLong": "4"}
+          command_name: "find"
+          database_name: *database_name
+      -
+        command_succeeded_event:
+          reply:
+            ok: 1.0
+            cursor:
+              id: { $numberLong: "42" }
+              ns: *namespace
+              firstBatch:
+                - { _id: 1, x: 11 }
+                - { _id: 2, x: 22 }
+                - { _id: 3, x: 33 }
+          command_name: "find"
+      -
+        command_started_event:
+          command:
+            getMore: { $numberLong: "42" }
+            collection: *collection_name
+            batchSize: { $numberLong: "1" }
+          command_name: "getMore"
+          database_name: *database_name
+      -
+        command_succeeded_event:
+          reply:
+            ok: 1.0
+            cursor:
+              id: { $numberLong: "42" }
+              ns: *namespace
+              nextBatch:
+                - { _id: 4, x: 44 }
+          command_name: "getMore"
+      -
+        command_started_event:
```

Comment:
I commented on this below, but just wanted to note that he Java driver fails this assertion.  Instead of sending a final getMore in order to exhaust the cursor, it notices that it has reached the limit and instead kills the cursor.

Reply:
I tried to get this to pass in Java driver and the changes just got more and more intrusive.  I think we should discuss.

Reply:
Ah, I was afraid there would be drivers that send `killCursors` instead of an additional `getMore`. I could relax the spec language to require closing the cursor, either by sending off an additional `getMore`, or by sending `killCursors`. This would document the behaviour, instruct drivers to ensure cursors are closed, but testing would be more difficult. I can see if I can cover both behaviours with a single test, but we may have to either not have any tests, not assert a whole lot about the additional command being sent (other than it being successful), or have two tests (one with getMore, one with killCursors) with drivers having to skip the one that doesn't apply to them.

Reply:
I think a goal here is to not require any changes to existing cursor logic, since the current logic, whatever it may be, should still be correct with 5.0.  I'd be ok not testing with 5.0 and defer spec/test changes to DRIVERS-1448.

Reply:
So essentially:
* we limit the test to <= 4.4
* we clarify the spec to instruct drivers to close cursors (using {{getMore}} or {{killCursors}})
* creating tests and changing the specification (e.g. by dropping the driver-side limit logic) are deferred to DRIVERS-1448

What do you think @jmikola?

Reply:
SGTM.

Reply:
I've made the changes as suggested above.